comixed-rest-api/src/main/java/org/comixed/controller/library/LibraryController.java

```diff
@@ -166,4 +166,20 @@ public ClearImageCacheResponse clearImageCache() {
 
     return new ClearImageCacheResponse(true);
   }
+
+  @PostMapping(
+      value = "/library/move",
+      produces = MediaType.APPLICATION_JSON_VALUE,
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  public MoveComicsResponse moveComics(@RequestBody() MoveComicsRequest request) {
```

Comment:
It would be, yes. I think we've got a mixed bag of final and non-final parameters in the codebase. We should probably have an issue created to do this across the board. I want to add REST Assured unit tests to the REST APIs and I think we could standardize things as a part of that effort.

Reply:
Yes it's good idea.